internal/builtinfunctions/functions.go

```diff
@@ -550,7 +550,10 @@ func getBindConstantsFunction() schema.CallableFunction {
 		schema.NewDisplayValue(
 			schema.PointerTo("Bind Constants"),
 			schema.PointerTo(
-				"Creates a list of objects with ID `CombinedObject`. "+
+				"Returns a list of objects each containing two properties: an item property which contains "+
+					"the corresponding item from the list in the first parameter; and, a constant property which "+
+					"contains the value of the second input parameter. "+
+					fmt.Sprintf("The two names are joined with the delimiter %q. ", CombinedObjIDDelimiter)+
```

Comment:
This jumps the rails: you've implicitly switched the context from talking about the contents of the list to the generated *id* of the list. And then we go *back* to talking about the object properties. Yeah, we can't really defer the description through a link in this context, which is a shame; but I think this should be pushed to the end of the description, perhaps with something like "The new object's ID is constructed by ..."

Reply:
yeah I just copy pasted the definition from the website

Reply:
I made a change